app/api/v1/endpoints/websocket.py

```
@@ -0,0 +1,28 @@
+    await manager.connect(websocket, user_id)
+    try:
+        while True:
+            # Keep connection alive and handle incoming messages
+            data = await websocket.receive_json()
+            
+            # Handle subscription requests
+            if data.get("type") == "subscribe":
+                # Handle subscription logic here
+                pass
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)